modules/softwarelandscape/exercise1.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Study & present
+type: exercise
+order: 2
+---
+
+## Exercise: Study and present (20 - 30 minutes)
```

Comment:
This is quite a nice idea for an exercise as we might also get to hear views from participants who have interacted in these spaces as part of their job already.